msvc/sdk-build.props

```diff
@@ -90,7 +92,7 @@
       <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
       <RuntimeLibrary Condition="'$(Configuration)'=='Debug'">MultiThreadedDebugDLL</RuntimeLibrary>
       <SystemHeaderDeps>true</SystemHeaderDeps>
-      <AdditionalOptions>-Werror -Wno-microsoft %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions>-Werror -Wno-microsoft -Wno-c++1z-extensions -Wno-ignored-attributes -Wno-deprecated-declarations %(AdditionalOptions)</AdditionalOptions>
```

Comment:
I'm worried that adding `-Wno-deprecated-declarations` will come back to bite us, since we use deprecation internally to signify unimplementedness.

If it doesn't warn, we will lose compiler warnings (and therefore errors) when we try to use something we have ourselves not implemented.